services/skus/radom/radom.go

```diff
@@ -0,0 +1,74 @@
+package radom
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/brave-intl/bat-go/libs/clients"
+)
+
+type Client struct {
+	client    *clients.SimpleHTTPClient
+	authToken string
+}
+
+func New(srvURL, authToken string) (*Client, error) {
+	cl, err := clients.New(srvURL, "")
+	if err != nil {
+		return nil, err
+	}
+
+	return &Client{client: cl, authToken: authToken}, nil
+}
+
+type Gateway struct {
+	Managed Managed `json:"managed"`
+}
+
+type Managed struct {
+	Methods []Method `json:"methods"`
+}
+
+type Method struct {
+	Network string `json:"network"`
+	Token   string `json:"token"`
+}
+
+type LineItem struct {
+	ProductID string `json:"productId"`
+}
+
+type Metadata struct {
+	Key   string `json:"key"`
+	Value string `json:"value"`
+}
+
+type CheckoutSessionRequest struct {
+	LineItems  []LineItem `json:"lineItems"`
+	Gateway    Gateway    `json:"gateway"`
+	SuccessURL string     `json:"successUrl"`
+	CancelURL  string     `json:"cancelUrl"`
+	Metadata   []Metadata `json:"metadata"`
+	ExpiresAt  int64      `json:"expiresAt"` // in unix seconds
+}
+
+type CheckoutSessionResponse struct {
+	SessionID  string `json:"checkoutSessionId"`
+	SessionURL string `json:"checkoutSessionUrl"`
+}
+
+func (c *Client) CreateCheckoutSession(ctx context.Context, creq CheckoutSessionRequest) (CheckoutSessionResponse, error) {
```

Comment:
The request is now passed by pointer, but the original comment suggested returning the result by pointer as well. What do you think about it (asking because that has not changed)?

Reply:
The `CheckoutResponse` has been implemented to behave like a value object so I was going to leave that.